django/urls/base.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import unicode_literals
```

Comment:
Is it possible to move this file exactly so that it appears as a moved file in git and we don't lose all the git blame info? Feel free to make changes in a subsequent commit (looks like you removed the docstring, for example)